src/pages/my_page/user_profile/MyProfile.tsx

```diff
@@ -74,140 +91,268 @@ export const MyProfile = () => {
       <Helmet>
         <title>{t('my_page.my_profile.user_profile')}</title>
       </Helmet>
+
       <Box
         sx={{
-          display: 'grid',
-          gridTemplateColumns: {
-            xs: '1fr',
-            md: '3fr 1fr',
-          },
-          columnGap: '1rem',
-          gridTemplateAreas: {
-            xs: '"user-profile" "user-identity" "research-profile"',
-            md: '"user-profile research-profile" "user-identity research-profile" ',
-          },
+          display: 'flex',
+          flexDirection: 'column',
+          bgcolor: 'secondary.main',
+          justifyContent: personQuery.isLoading ? 'start' : 'space-between',
         }}>
-        <BackgroundDiv
-          sx={{
-            bgcolor: 'info.light',
-            gridArea: 'user-profile',
-          }}>
-          <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1.5rem' }}>
-            <Typography variant="h2" id="personalia-id">
-              {t('my_page.my_profile.heading.personalia')}
-            </Typography>
-            {personQuery.isLoading && !person ? (
-              <PageSpinner aria-labelledby="personalia-id" />
-            ) : (
-              <>
-                <Typography>{t('my_page.my_profile.user_profile_description')}</Typography>
-                <Formik initialValues={initialValues} onSubmit={updatePerson} enableReinitialize>
-                  {({ isSubmitting, dirty, resetForm }: FormikProps<CristinPersonFormData>) => (
-                    <Form>
+        <Typography variant="h2" sx={{ margin: '1rem' }}>
+          {t('my_page.my_profile.heading.personalia')}
+        </Typography>
+
+        {personQuery.isLoading && !person ? (
+          <PageSpinner aria-labelledby="personalia-id" />
+        ) : (
+          <Formik
+            initialValues={initialValues}
+            validationSchema={personaliaValidationSchema}
+            onSubmit={updatePerson}
+            enableReinitialize>
+            {({ isSubmitting, dirty, resetForm, values, isValid }: FormikProps<CristinPersonFormData>) => (
+              <Form>
+                <Box
+                  sx={{
+                    display: 'grid',
+                    gridTemplateColumns: { xs: '1fr', md: '2fr 1fr' },
+                    columnGap: '2rem',
+                    ml: '1rem',
+                  }}>
+                  <Grid container gridColumn={1} rowGap={1} columns={16}>
+                    <Grid item xs={16} md={3}>
+                      <StyledTypography>{t('my_page.my_profile.person_name')}</StyledTypography>
+                    </Grid>
+                    <Grid item xs={14} md={12}>
+                      <Box
+                        sx={{
+                          height: '100%',
+                          display: 'grid',
+                          gridTemplateColumns: '1fr 1fr',
+                          gap: '1rem',
+                          justifyContent: 'space-evenly',
+                          alignItems: 'center',
```

Comment:
Ser ut som dette gjør samme nytte?
```suggestion
                          display: 'grid',
                          gridTemplateColumns: '1fr 1fr',
                          gap: '1rem',
```